modules/ROOT/pages/proc_deploying-an-application-to-openshift.adoc

```diff
@@ -8,38 +8,17 @@
 [discrete]
 === Prerequisites
 
+* You have access to OpenShift Container Platform cluster.
```

Comment:
Considering that we are removing the entire CRC section, please add a line similar to the one in the installing Pipelines operator section

Reply:
Could you please fix the link on line 12 too? Does not work for me.